### ISSUE TITLE
About the metrics

### ISSUE THREAD

**Issue by zhuhrun** [hq-King/Affordance-R1#3]
Hello, excellent work.

I seem to be encountering some issues. I trained the model on an A800-80GB GPU. 
During my training, the metric `“critic/rewards/mean”` appears difficult to raise after reaching a certain value. I'm curious to know what values the metrics `“critic/rewards/mean”` and `“critic/rewards/min”` typically reach in your training?May i have your training log.

```python
# reward
"critic/rewards/mean": torch.mean(sequence_reward).detach().item(),
"critic/rewards/max": torch.max(sequence_reward).detach().item(),
"critic/rewards/min": torch.min(sequence_reward).detach().item(),
``` 

Thank you for your prompt reply.

**Comment by hq-King** [comment]
Sorry for my late response, I have uploaded one picture about this, and hope it will be helpful

<img width="2776" height="1518" alt="Image" src="https://github.com/user-attachments/assets/8d572897-a28a-4b43-b5dd-a50781423720" />
